src/index.d.ts

```diff
@@ -5170,6 +5203,138 @@ declare namespace admin.messaging {
   }
 }
 
+declare namespace admin.machineLearning {
+    /**
+     * Interface representing options for listing Models.
+     */
+    interface ListModelOptions {
+      listFilter?: string;
+      pageSize?: number;
+      pageToken?: string;
+    }
+
+    /** Response object for a listModels operation. */
+    interface ListModelsResult {
+      readonly models: Model[];
+      readonly pageToken?: string;
+    }
+
+    /**
+     * A TFLite Model output object
+     */
+    interface TFLiteModel {
+      readonly sizeBytes: number;
+
```

Comment:
Remove redundant space. Here and below.

Reply:
done

Reply:
I was actually referring to the redundant newlines between entries. Sorry for not being explicit.